MAUI/RadioButton/Event.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: Event in .NET MAUI Radio Button control | Syncfusion
+description: Learn about Event support in Syncfusion Essential Studio .NET MAUI Radio Button control, its elements and more.
+platform: Maui
+control: Radio Button
+documentation: ug
+---
+
+# Event in .NET MAUI Radio Button (SfRadioButton)
+
+## StateChanged event
+
+Occurs when the value(state) of the [`IsChecked`] property is changed by either touching the check box or setting the value to the [`IsChecked`] property using XAML or C# code. The event arguments are of type [`StateChangedEventArgs`] and expose the following property:
```

Comment:
This event occurs when the value (state) of the `IsChecked` property is changed by touching the check box or setting the value to the `IsChecked` property using XAML or C# code. The event arguments are of type `StateChangedEventArgs` and expose the following property: